resources/traits.yaml

```diff
@@ -810,22 +815,28 @@ traits:
   - Knative
   - OpenShift
   description: 'The Mount trait can be used to configure volumes mounted on the Integration
-    Pod. nolint: tagliatelle'
+    Pods. nolint: tagliatelle'
```

Comment:
Fixed in #2851 - if you rebase, you should have it fixed when regenerating the documentation

Reply:
It's already rebased, but it's not fixing it locally.

Reply:
Yeah, I've spotted the problem and fixed in #2873